run_intermediate_expansion.py

```diff
@@ -8,12 +8,23 @@
 score_func, cache_stats, reset_cache = create_cached_func(uncached_score_func)
 from common_interactive import diffprompt
 
-from prompts import prompt, expansion_count, min_lines, check_func
+from prompts import prompt
 from common import limit_depth, max_completion_depth
 from common_stats import stats
 
 import llm
 
+min_lines = 5
+expansion_count = None
+
+def check_func(v):
```

Comment:
add a TODO that this is not the right place for this?

Reply:
sure

Reply:
actually should I go ahead and move this to prompts.py? Or do we want to think further about where it should go?

Reply:
this will unfortunately also break the expansion with the prompts problem, i think. i think  you should define it in prompt and set the prompt to a dummy clover thing? anyways, let me merge and we can fix later.